### PR TITLE
If any subcommand of pseudo source fails - exit with error

### PR DESCRIPTION
**Pull request by edmundreinhardt**
Pseudo source consists of multiple CL commands in a stream file.

When

### CODE REVIEW

**Review by irfanshar — APPROVED** [review]

**Comment by irfanshar** [comment]
Perhaps we should update  bob-recursive-example/QBNDSRC/SAMPLE.BNDDIR to have an example of this usage.

**Comment by edmundreinhardt** [comment]
> Perhaps we should update bob-recursive-example/QBNDSRC/SAMPLE.BNDDIR to have an example of this usage.



> Perhaps we should update bob-recursive-example/QBNDSRC/SAMPLE.BNDDIR to have an example of this usage.

I agree we should do that, but not before we release this in a new version of Bob, because it will break everyone out there
What I will do is release it on a branch that we can merge at that time

**Comment by edmundreinhardt** [comment]
@irfanshar please resolve the conflicts and merge?

**Comment by edmundreinhardt** [comment]
Actually hold off, let;s do this with Marius

File: src/scripts/extractAndLaunch [hunk 1]
```diff
@@ -23,7 +29,11 @@ while read -r line; do
   if [ -n "$cmd" ]; then
     cmd=$(echo "$cmd" | sed "s/&O/$objlib/g" | sed "s/&N/$objname/g")
     "$SCRIPT_DIR"/launch "$joblog_json" "$cmd" </dev/null
-    # echo "Running $cmd"
+    if [ $? -ne 0 && suppressError -ne "Y"]; then
```
Comment (ITMentalist) [inline src/scripts/extractAndLaunch:32]: @irfanshar  is this the correct place to look for both error conditions?

Comment (irfanshar) [reply src/scripts/extractAndLaunch:32]: Yes this is

**Review by ITMentalist — APPROVED** [review]
I am good with the suggested changes

File: docs/welcome/features.md [hunk 2]
```diff
@@ -59,8 +59,15 @@ Note:
 
 The following object types are supported as pseudo-source.  
 The CL command to create the object is stored in a file with the given extension.  
+
 Note that for *MSGF and *BNDDIR, the CL commands should include the delete of the MSGF and BNDDIR before creating the new one in order to avoid the timestamp getting old and breaking the make processing.
 
+Normal processing of commands will continue until an error is reached unless prefixed with a `!` in front of the command. 
+
+Usage the prefix of `!` will ignore any errors for the command. 
+ex. `!DLTBNDDIR XXXX/YYYYY` will then ignore the error if the Binding Directory doesnt exist and continue processing.
```
Comment (irfanshar) [inline docs/welcome/features.md:68]: Can we replace doesnt with doesn't?

**Review by irfanshar — CHANGES_REQUESTED** [review]

File: docs/welcome/features.md (hunk 2 above)
Comment (ITMentalist) [reply docs/welcome/features.md]: cool , resolved the grammar error.

**Review by irfanshar — APPROVED** [review]

**Pull request merged**